### PR TITLE
Merge changes from features/employees to develop

### PR DESCRIPTION
**Pull request by Luwat**
Kindly, review changes made to the employees screen, I am currently using a placeholder data to display the employees list.

### CODE REVIEW

File: app/(tabs)/employees.tsx [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
+        keyExtractor={(employee) => employee.id.toString()}
+        renderItem={(employee) => (
+          <Employee
+            firstName={employee.item.firstName}
+            lastName={employee.item.lastName}
+            position={employee.item.position}
+          />
+        )}
+        ListHeaderComponent={() => <EmployeesListHeader />}
+      />
```
Comment (Oludefiyinfoluwa06) [inline app/(tabs)/employees.tsx:23]: I think you should have the `ListEmptyComponent` in the case that there are no employees

File: lib/definitions.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export type EmployeesData = {
+    id: number;
```
Comment (Oludefiyinfoluwa06) [inline lib/definitions.ts]: The type of the id that will be coming from the database is going to be a string not a number

File: app/(tabs)/employees.tsx (hunk 1 above)
Comment (Luwat) [reply app/(tabs)/employees.tsx:23]: Okay...!  I will include that now. I suppose that will indicate that there is no employee in the list. Thanks

File: lib/definitions.ts (hunk 2 above)
Comment (Luwat) [reply lib/definitions.ts]: Okay, I will also adjust that.

Comment (mykoman) [reply lib/definitions.ts]: You can use 
id: number | string

**Review by mykoman — APPROVED** [review]
Well done.
I left a comment

**Pull request merged**